docs/40-troubleshooting.md

```diff
@@ -1,4 +1,64 @@
-# Getting logs without Azure Portal
+## Debugging and Troubleshooting
+
+### After creating my cluster, my nodes go to a 'startTaskFailed' state. Why?
+The most common case for this is that there was an issue with package installation or the custom script failed to run. To troubleshoot this you can simply download the output logs from the node.
+
+The following 2 nodes failed while running the start task:
+tvm-769611554_1-20170912t183413z-p
+tvm-769611554_2-20170912t183413z-p
+
+
+```r
+cluster <- makeCluster('myConfig.json')
+
+# Often you will see an error printed to the console such as:
+#  The following 2 nodes failed while running the start task:
+# tvm-769611554_1-20170912t183413z-p
+# tvm-769611554_2-20170912t183413z-p
+
+# If you do not get this message, you can list the Nodes in the cluster
+# Look for the node$id values, for example:
+# $value[[1]]$id
+# [1] "tvm-769611554_1-20170912t183413z-p"
+rAzureBatch::listPoolNodes('<CLUSTER_ID>')
+
+# Get standard error file
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-errors.txt")
+
+# Get standard log file
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-logs.txt")
+```
+
+### My job never starts running. How can I troubleshoot this issue?
+This is often caused by the node not being in a good state. Take a look at the state of the nodes in the cluster to see if any of them are have an error or are in a failed state. If the node is in a startTaskFailed state follow the instructions above. If the node is in an 'unknown' or 'unusable' state you may need to manually reboot the node.
```

Comment:
How should they take a look at the state of the nodes in the cluster? if they are using the portal, we may as well tell them that they can reboot their node from there too.

Also, i think people will think "if i reboot my node, won't that mess up the deployment?" - we can give them some peace of mind by saying "When rebooting the node, we will run the necessary scripts to get your node setup to work for you doAzureParallel cluster"